test/SymbolGraph/Symbols/Mixins/Availability/Inherited/IntroducedReplaced.swift

```diff
@@ -0,0 +1,42 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift %s -module-name IntroducedReplaced -emit-module -emit-module-path %t/
+// RUN: %target-swift-symbolgraph-extract -module-name IntroducedReplaced -I %t -pretty-print -output-dir %t
+// RUN: %FileCheck %s --input-file %t/IntroducedReplaced.symbols.json --check-prefix=GREATERTHAN
+// RUN: %FileCheck %s --input-file %t/IntroducedReplaced.symbols.json --check-prefix=EQUAL
+
+// REQUIRES: OS=macosx
+
+@available(macOS, introduced: 10.15)
+public struct S {
+  // Not possible: declarations cannot be more available than an enclosing scope
+  // @available(macOS, introduced: 10.10)
+  // public func lessThan() {}
+
+  @available(macOS, introduced: 10.16)
+  public func greaterThan() {}
+
+  @available(macOS, introduced: 10.15)
+  public func equal() {}
```

Comment:
Can we also add tests for when a symbol's transitive parent has an availability annotation? Something like:

```swift
@available(macOS, introduced: 10.15)
public struct S {
  public struct T {
    public func foo() {}
  }
}
```

Reply:
Ah yes, I have some tests for those but forgot to add them to this commit. Will add them.